docs/integrations/litestar.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: Starlite
+title: Litestar
 ---
 
-# Starlite
+# Litestar
 
 Strawberry comes with an integration for
-[Starlite](https://starliteproject.dev/) by providing a
+[Litestar](https://litestar.dev/) by providing a
 `make_graphql_controller` function that can be used to create a GraphQL
 controller.
 
-See the example below for integrating Starlite with Strawberry:
+See the example below for integrating Litestar with Strawberry:
 
 ```python
 import strawberry
-from starlite import Starlite
-from strawberry.starlite import make_graphql_controller
+from litestar import Litestar
+from strawberry.litestar import make_graphql_controller
```

Comment:
oh, we haven't updated the code yet, if you wanna do that to we should do an alias from the old name and do a deprecation, I'm also fine with just updating the docs for now

Reply:
I've got to switch to $dayjob now. I  might be able to get to that this weekend but I'm happy for you to add the necessary changes here or merge this and add the changes in a follow-up PR.